app/helpers/application_helper.rb

```diff
@@ -49,27 +51,20 @@ def icon(name, pixels = "20")
   end
 
   def icon_url(name)
-    prefix = "https://res.cloudinary.com/practicaldev/image/upload/"
-    case name
-    when "twitter"
-      prefix + "v1456342401/twitter-logo-silhouette_1_letrqc.png"
-    when "github"
-      prefix + "v1456342401/github-logo_m841aq.png"
-    when "link"
-      prefix + "v1456342401/link-symbol_apfbll.png"
-    when "volume"
-      prefix + "v1461589297/technology_1_aefet2.png"
-    when "volume-mute"
-      prefix + "v1461589297/technology_jiugwb.png"
-    else
-      prefix + "v1456342953/star-in-black-of-five-points-shape_sor40l.png"
-    end
+    postfix = {
+      'twitter'     => 'v1456342401/twitter-logo-silhouette_1_letrqc.png',
+      'github'      => 'v1456342401/github-logo_m841aq.png',
+      'link'        => 'v1456342401/link-symbol_apfbll.png',
+      'volume'      => 'v1461589297/technology_1_aefet2.png',
+      'volume-mute' => 'v1461589297/technology_jiugwb.png'
+    }.fetch(name, 'v1456342953/star-in-black-of-five-points-shape_sor40l.png')
+
+    "https://res.cloudinary.com/practicaldev/image/upload/#{postfix}"
   end
 
   def cloudinary(url, width = nil, _quality = 80, _format = "jpg")
-    if Rails.env.development? && (url.blank? || !url.include?("http"))
-      return url
-    end
+    return if Rails.env.development? && (url.blank? || url.exclude?("http"))
+
```

Comment:
Doesn't this change the behavior? It returned "url" before, not it returns nil

Reply:
Oops